flowlogs_reader/flowlogs_reader.py

```diff
@@ -97,6 +99,16 @@ class FlowRecord:
     ]
 
     def __init__(self, event_data, EPOCH_32_MAX=2147483647):
+        # AWS allows VPCFlowLogs and TransitGateway logs to exist in the same file. 
+        # In this case, we will set an invalid version number so that reader classes 
+        # can properly skip that record. 
+        # https://docs.aws.amazon.com/vpc/latest/tgw/tgw-flow-logs.html#flow-log-records
+        if event_data.get('account_id') in TGW_FAIL:
```

Comment:
`TGW_FAIL` is only used in this location and is a confusing variable name, IMO. I'd suggest either:

- Removing the variable and just using a `set` here for checking membership
- Renaming the variable to `TGW_ACCOUNT_ID` or similar